packages/pluggableWidgets/line-chart-web/rollup.config.js

```diff
@@ -0,0 +1,93 @@
+import { join } from "path";
+import replace from "rollup-plugin-re";
+import { nodeResolve } from "@rollup/plugin-node-resolve";
+import { getBabelInputPlugin, getBabelOutputPlugin } from "@rollup/plugin-babel";
+import commonjs from "@rollup/plugin-commonjs";
+import { terser } from "rollup-plugin-terser";
+
+export default args => {
+    const production = Boolean(args.configProduction);
+    const outDir = join(__dirname, "dist/tmp/widgets/com/mendix/shared");
+    const result = args.configDefaultConfig;
+    const [jsConfig, mJsConfig] = result;
+
+    // We define the new library externals for the entry points
+    const libraryExternals = [/^react-plotly\.js($|\/)/, /(^|\/)shared\/api.js$/];
```

Comment:
Should't `plotly.js` also be part of this (it's in the dependencies)? 🤔

Reply:
Nope, because its being imported inside the react-plotly, which already bundles it.